src/pages/dev/send-tokens/interchain-tokens/upgrade-tokens.mdx

```diff
@@ -6,6 +8,10 @@ The Interchain Token Service is deployed to [`0xB5FB4BE02232B1bBA4dC8f81dc24C269
 
 Now, let's explore how to transform existing tokens into Interchain Tokens.
 
+<Callout emoji="🚨">
+  **NOTE:** The security of your token is limited to the security of the chains it integrates with. Since blockchains can have different security practices, we recommend verifying each source chain and source contract address before integration.
```

Comment:
Perhaps "as part of your contract"? 

The problem in this context isn't the source chain/address in your `__execute` method, although that's important, but your chain choices should be built understanding the security of each chain you choose to deploy on.